apps/api/migrations/082_add_savings_schedules_onchain_mandate_id.down.sql

```
@@ -0,0 +1,4 @@
+-- Remove on-chain mandate ID column from savings schedules
+
+DROP INDEX IF EXISTS idx_savings_schedules_onchain_mandate_id;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)